models/biswap/bnb/biswap_bnb_trades.sql

```diff
@@ -12,7 +12,7 @@
     )
 }}
 
-{% set project_start_date = '2021-03-15' %}
+{% set project_start_date = '2021-05-22' %}     -- select min(evt_block_time) from biswap_bnb.BiswapPair_evt_Swap
```

Comment:
Hi @developerfred , I change the project start date based on [the query result](https://dune.com/queries/1647951?d=1). Please verify if any trade record I may miss with this change.

Reply:
Ok